Populated dropdown

Faithy4444 opened the pull request:
<!--

You must title your PR like this:

COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK

For example,

NW4 | Carol Owen | HTML-CSS-Module | Week1

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

EvelynMwanunura commented:
Hi Faith, Could you please review the following updates for the HTML file?

The <title> tag is missing from the <head> section.

The <form> element is currently placed outside the <body> tag.

Kindly add a <div>  to display the user’s agenda information.

The <label for> attribute for the topic input should match the input’s id (topicName).

Faithy4444 commented:
I added a title tag, 
what element is outside what tag?
the users information will be displayed in the 'agenda' div and the rest of the elements will be created using javascript
I fixed the name and the id of topic name and it should be in the second pull request I made yesterday(if not it will be in todays PR)

EvelynMwanunura commented:
I realised my previous comment didn’t show the full text because of the formatting I used. I’ve edited the comment now to fix that.

Faithy4444 commented:
alrighty. The fixes you mentioned should be in the second PR, if not then they will be in the next one because I made those changes yesterday

EvelynMwanunura commented:
Noted, thanks.
